src/Raven.Client/Documents/Conventions/DocumentConventions.cs

```diff
@@ -54,6 +54,40 @@ public class DocumentConventions : Client.Conventions
 
         public readonly BulkInsertConventions BulkInsert;
 
+        public readonly AggressiveCacheConventions AggressiveCache;
+
+        public class AggressiveCacheConventions
+        {
+            private readonly DocumentConventions _conventions;
+            private readonly AggressiveCacheOptions _aggressiveCacheOptions;
+            
+            public AggressiveCacheConventions(DocumentConventions conventions)
```

Comment:
should be internal

Reply:
Fixed in https://github.com/ravendb/ravendb/pull/9179